src/main/scala/com/yotpo/metorikku/input/file/FileReader.scala

```diff
@@ -38,13 +40,33 @@ object FileReader {
     }
   }
 
+  def getSchema(path: String, spark: SparkSession): StructType = {
```

Comment:
There is already a logic for this in the JSON table reader. You only want to add the ability to specify a path for it instead of guessing it.

Reply:
Also support this in all readers (add this to filereader trait)